doc/input-output-reference/src/overview/group-schedules.tex

```diff
@@ -590,13 +587,18 @@ \subsubsection{Outputs}\label{outputs-031}
 Output:Variable,Activity_Shopping_Sch,Schedule Value,hourly;
 \end{lstlisting}
 
+A wild card $*$ would also work in place of the schedule name. For example, the following syntax would report all the schedule values for all the schedules in an IDF file:
+\begin{lstlisting}
+Output:Variable,*,Schedule Value,hourly;
+\end{lstlisting}
+
 \subsection{Schedule:File:Shading}\label{schedulefileshading}
```

Comment:
Another way of using output:variable documented and referenced properly in other places.